### ISSUE TITLE
Request to Improve or Configure the New Hash Feature

### ISSUE THREAD

**Issue by mulder999** [umami-software/umami#3081]
### Describe the feature or enhancement

I would like to request the ability to disable or configure the new hash feature. Using hashes in SPAs is generally not considered best practice. Alternatively, improving the UI to align with standard page semantics would be a great enhancement.

Thank you for considering this feedback!

**Comment by mikecao** [comment]
You're right, some people may not want to track hashes at all. We'll add a way to disable it.

**Comment by mikecao** [comment]
Fixed in v2.16.0.

**Comment by mulder999** [comment]
@mikecao thanks for the fix. However it is still counting each navigation within the page as a page visit.
